assets/rln-relay/rln-relay.msc

```diff
@@ -0,0 +1,15 @@
+# MSC for some fictional process
+msc {
+  hscale = "2";
+
+  a [label = "Relay Node"],b [label = "Membership Contract"];
```

Comment:
I see PR is in draft, but it would be nice to add a second Relay node in case this isn't planned already